### PR TITLE
NOTIF-620 behavior group ? icon collapsed when clicked

### PR DESCRIPTION
**Pull request by aferd**
https://user-images.githubusercontent.com/86322239/173100796-3594268d-9119-419f-a988-990dba8aeef2.mov

### CODE REVIEW

**Review by josejulio — CHANGES_REQUESTED** [review]
Other applications and IIRC the designs uses the `Popover`-style (click to show).

Could you try the following?
- Control the state with the `isVisible` prop on the popover? 
-  Use the `shouldClose` / `shouldOpen` functions - they receive an event. Calling `event.stopPropagation();` should stop the event from reaching the expanded button.

Something else that could work is to rearrange the components. 

Right now we have
```tsx
<ExpandingToggle>
  <Split>
    <SplitItem>Tittle</SplitItem>
    <SplitItem>PossibleButton</SplitItem>
    <SplitItem>Popover</SplitItem>
  </Split>
</ExpandingToggle>
```

Maybe it could look good and work well with:
```tsx
  <Split>
    <ExpandingToggle><SplitItem>Tittle</SplitItem></ExpandingToggle>
    <SplitItem>PossibleButton</SplitItem>
    <SplitItem>Popover</SplitItem>
  </Split>
```

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/173430442-d9691cae-5001-4f32-8938-da9607696843.mov

**Comment by josejulio** [comment]
It looks to me like there is an extra space between "Behavior groups" and the `(6)` that wasn't there. Could you check?

**Comment by mmenestr** [comment]
Yes, I agree there's too much space between the two - should be 16px only

**Comment by aferd** [comment]
Screen Shot 2022-06-13 at 5 06 07 PM

**Review by josejulio — APPROVED** [review]
lgtm

**Comment by mmenestr** [comment]
Just one small thing - all actionable icons should be grey by default, and black only on hover and on click! So in the screenshot, the question circle should be - pf-global--color--200


image

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/173669147-dabc96bd-b98f-4a36-8c8b-52d2b83877e0.mov

**Comment by aferd** [comment]
@mmenestr changed to default color then changes when hover/clicked

File: src/pages/Notifications/List/BehaviorGroupsSection.tsx [hunk 1]
```diff
@@ -55,6 +55,17 @@ const titleClassName = style({
     color: global_palette_black_1000.var
 });
 
+const defaultIconClassName = style({
+    marginTop: '2px',
+    color: global_Color_200.var,
```
Comment (josejulio) [inline src/pages/Notifications/List/BehaviorGroupsSection.tsx]: Is this needed? I though it would work out of the box without this color.

**Review by josejulio — APPROVED** [review]
lgtm

File: src/pages/Notifications/List/BehaviorGroupsSection.tsx (hunk 1 above)
Comment (aferd) [reply src/pages/Notifications/List/BehaviorGroupsSection.tsx]: ah, nope. not needed, thanks! I did not try without it. It's been removed.

**Comment by mmenestr** [comment]
Looks good thank you!

**Pull request merged**